app/views/book-an-appointment/confirm-appointment.html

```diff
@@ -24,7 +24,7 @@ <h1 class="heading-xlarge">
       appointment_length: appointment.appointment_length,
       appointment_type: appointment.appointment_type,
       appointment_type_class: appointment.appointment_type_class,
-      address: practice.name + '<br>' + practice.address.join('<br>')
+      address: appointment.address.join('<br>')
```

Comment:
It currently "works" (by luck) since the `appointment_summary` macro helpfully defaults the address to Lakeside surgery - but I don't think that's ideal.